index.bs

```diff
@@ -579,38 +589,43 @@ When this method is invoked, the user agent MUST execute the following algorithm
         such as [=domain=], [=ipv4 address=], [=ipv6 address=], [=opaque host=], or [=empty host=].
         Only the [=domain=] format of [=host=] is allowed here.
 
-1. Let |rpId| be |effectiveDomain|.
-
 <!-- Note: this next step is actually a top-level step, but bikeshed wanted it indented this much in order to compile w/o errors
 -->
     <li id='CreateCred-DetermineRpId'>
-       If |options|.{{MakePublicKeyCredentialOptions/rp}}.{{PublicKeyCredentialEntity/id}} is [=present=]:
 
-        1. If |options|.{{MakePublicKeyCredentialOptions/rp}}.{{PublicKeyCredentialEntity/id}} [=is not a registrable domain suffix of 
-            and is not equal to=] |effectiveDomain|, return a {{DOMException}} whose name is "{{SecurityError}}", and terminate
-            this algorithm.
+        If <code>|options|.{{MakePublicKeyCredentialOptions/rp}}.{{PublicKeyCredentialEntity/id}}</code>
+            <dl class="switch">
 
-        1. Set |rpId| to |options|.{{MakePublicKeyCredentialOptions/rp}}.{{PublicKeyCredentialEntity/id}}.
+                :   Is [=present=]
+                ::  If <code>|options|.{{MakePublicKeyCredentialOptions/rp}}.{{PublicKeyCredentialEntity/id}}</code> [=is not a
+                    registrable domain suffix of and is not equal to=] |effectiveDomain|, return a {{DOMException}} whose name
+                    is "{{SecurityError}}", and terminate this algorithm.
 
-            Note: |rpId| represents the caller's [=RP ID=]. The [=RP ID=] defaults to being the caller's [=environment settings
-            object/origin=]'s [=effective domain=] unless the caller has explicitly set
-            |options|.{{MakePublicKeyCredentialOptions/rp}}.{{PublicKeyCredentialEntity/id}} when calling
+                :   Is [=present|not present=]
+                ::  Set <code>|options|.{{MakePublicKeyCredentialOptions/rp}}.{{PublicKeyCredentialEntity/id}}</code> to
+                    |effectiveDomain|.
+            </dl>
+
+        Note: <code>|options|.{{MakePublicKeyCredentialOptions/rp}}.{{PublicKeyCredentialEntity/id}}</code> represents the
+            caller's [=RP ID=]. The [=RP ID=] defaults to being the caller's [=environment settings object/origin=]'s
+            [=effective domain=] unless the caller has explicitly set
+            <code>|options|.{{MakePublicKeyCredentialOptions/rp}}.{{PublicKeyCredentialEntity/id}}</code> when calling
             {{CredentialsContainer/create()}}.
     </li>
 
-1. Let |normalizedParameters| be a new [=list=] whose [=list/items=] are pairs of {{PublicKeyCredentialType}} and
-    an {{AlgorithmIdentifier}}.
+1. Let |credTypesAndPubKeyAlgs| be a new [=list=] whose [=list/items=] are pairs of {{PublicKeyCredentialType}} and
+    a {{COSEAlgorithmIdentifier}}.
+
+1. [=list/For each=] |current| of <code>|options|.{{MakePublicKeyCredentialOptions/pubKeyCredParams}}</code>:
```

Comment:
I'd love if the additions of `<code>` were in a separate PR so I could focus on semantic changes in this one. It's not worth changing that now, but in the future please try.

Reply:
please see comment above.

Reply:
For these in particular, I'd be totally happy if you just merge them without review. I just don't want to see them when I'm trying to review something else.

Reply:
ok, well, sorry. i guess trying to rely on reviewing plain source html complicates things. and the [built-in diff up at the top in the orig post](https://s3.amazonaws.com/pr-preview/w3c/webauthn/cca20d3...a1b998f.html) does not diff against the current state of master (I just checked (it would be more helpful if it would)). That's why I generate rendered html diffs as needed (links below).

Reply:
Oh, also, in looking at the built-in github;com plain html side-by-side diff, it seems it is diffing against an older commit of master and so there's stuff in here that's not relevant (sorry), so using the rendered html diffs I generated (links below) are the way to go.